modules/oe_content_organisation/oe_content_organisation.post_update.php

```diff
@@ -18,3 +19,34 @@ function oe_content_organisation_post_update_00001(): void {
   $component['settings']['removed_reference'] = 'keep';
   $form_display->setComponent('oe_organisation_contact', $component)->save();
 }
+
+/**
+ * Enable new dependency.
+ */
+function oe_content_organisation_post_update_00002(): void {
+  \Drupal::service('module_installer')->install(['description_list_field']);
+}
+
+/**
+ * Create overview field.
+ */
+function oe_content_organisation_post_update_00003(): void {
+  $storage = new FileStorage(drupal_get_path('module', 'oe_content_organisation') . '/config/post_updates/00003_create_fields');
+  \Drupal::service('config.installer')->installOptionalConfig($storage);
+}
+
+/**
+ * Update entity form display.
+ */
+function oe_content_organisation_post_update_00004(): void {
```

Comment:
We did the same for other content types when we had new dependencies and fields as suggested by @ademarco in case something goes wrong we know which update failed.